teams/developer-relations/development-roadmap/roadmap-topic-breakdown.md

```diff
@@ -1,31 +1,31 @@
 # Roadmap Topic Breakdown
 
-## Topic: Python
+## Topic: NoSQL/MongoDB
 
-**Module: Intermediate Python**
+**Module: Intro to NoSQL/MongoDB**
 
 _**Learning Objectives**_
 
-* Students will continue applying their fundamental Python skills learned from the previous module.
-* Students will begin learning about how to integrate basic data structures like lists and dictionaries to add complexity to their Python code.
-* Students will learn how they can use for/while loops and if statements to control the flow of their code.
-* Students will learn how to represent strings in Python using F-strings.
+* Students will learn the main difference between NoSQL and SQL databases.
```

Comment:
additionally, there should be some learning objectives about MongoDB's various products - Stitch, Atlas, etc. The MongoDB workshop can provide more insight, and @jeffchhen could also help with this